Add meta description to some pages (#4534)
wazuh/wazuh-documentation: source/user-manual/kibana-app/features/settings.rst

.. Copyright (C) 2021 Wazuh, Inc.

.. meta::

>   :description: The Wazuh Kibana plugin gives you a quick view of your agents, alerts, and cluster. Learn more about the Settings page in this section. 

Reviewer: ```suggestion
  :description: The Wazuh Kibana plugin gives you a quick view of your agents, alerts, and cluster. Learn how to configure its features in this section. 
```

>   :description: The Wazuh Kibana plugin gives you a quick view of your cluster, agents and alerts. Learn more about the Settings page in this section. This page allows you to configure and customize your Wazuh app experience. 

Reviewer 2: ```suggestion
  :description: The Wazuh Kibana plugin gives you a quick view of your agents, alerts, and cluster. Learn more about the Settings page in this section. 
```
